### PR TITLE
Readme

### LINKED ISSUE #11: Create documentation
We need to have some basic documentation for this project. This will help explain how to work with the service for new team members a refresh the memory of anyone coming back to the project.

The documentation should include the following

- [] [ADR](https://github.com/joelparkerhenderson/architecture_decision_record) records
- [] Simple architecture diagram
- [] Deployment details - accounts / regions / concourse
- [] Important links / supporting resources

### PR DESCRIPTION
**Pull request by alice-carr**
Put some basic info in the readme
closes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Deployment details
+Merges to master will trigger a deploy in [Concourse](https://cd.gds-reliability.engineering/teams/cybersecurity-tools/pipelines/nessus).
+
+This deploys to account `676218256630` in region `eu-west-2`
+
+## Prerequisites
+Before working on this project, install pipenv and create an environment for this repo:
+```
+brew install pipenv
```
Comment (pritchyspritch) [inline README.md:21]: Should probably just put pip install pipenv so it works for everyone

Comment (alice-carr) [reply README.md:21]: good point, I'll do that now

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+cd cyber-security-nessus
+pipenv shell
```
Comment (pritchyspritch) [inline README.md:23]: If you're contributing to the project for the first time you will need to run `pipenv install --dev` (--dev installs all packages, as well as all dev packages such as pytest and black)

**Comment by pritchyspritch** [comment]
Maybe we should also add some info about how to add a new scheduled scan in the toml config?

**Pull request merged**